src/ResourceManager/Network/Commands.Network/help/Get-AzureRmHubVirtualNetworkConnection.md

```diff
@@ -0,0 +1,170 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
```

Comment:
Be sure to regenerate these after making changes to parameter sets and such.

Reply:
Yes - regenerated files as required.